beacon-chain/cache/active_indices.go

```diff
@@ -42,7 +42,7 @@ type ActiveIndicesCache struct {
 }
 
 // activeIndicesKeyFn takes the epoch as the key for the active indices of a given epoch.
-func activeIndicesKeyFn(obj interface{}) (string, error) {
+func ActiveIndicesKeyFn(obj interface{}) (string, error) {
```

Comment:
Do not make this public, make a new test file that is not blackbox testing if you need access to private functions.